src/web/app/page.component.ts

```diff
@@ -87,6 +90,9 @@ export class PageComponent {
         fromEvent(window, 'online').pipe(mapTo(true)),
         fromEvent(window, 'offline').pipe(mapTo(false)),
     );
+
+    // Closes modals when the back or forward button is clicked in the browser
+    location.onPopState(() => this.modalService.dismissAll());
```

Comment:
Do we have > 1 modal at any point of time? If we can use `.close()` instead of `.dismissAll()`.

If so let's change the comment from, <s>modals</s> -> modal

Reply:
> If we can use .close() instead of .dismissAll()

Wouldn't we have to implement this behaviour for every single modal in that case? `NgbModal` doesn't have a `.close()` function 😕 

Also, I think `.dismiss()` is better than `.close()` in our case. If you try to `.close()` the `tm-course-soft-deletion-confirm-modal` when going back to the previous page, the course ends up being deleted.

> If so let's change the comment from, modals -> modal

I've updated the comment 👍